packages/client-sdk-nodejs/src/config/configurations.ts

```diff
@@ -70,6 +70,14 @@ export class Laptop extends CacheConfiguration {
       transportStrategy: transportStrategy,
       middlewares: defaultMiddlewares,
       throwOnErrors: false,
+      channelConfiguration: {
+        maxSessionMemoryMB: grpcConfig.getMaxSessionMemoryMb(),
```

Comment:
mentioned this in slack but I'd rather put all the new configs into the existing `grpcConfig` instead of adding a new object/layer.